apps/src/studio/studio.js

```diff
@@ -5916,6 +5974,7 @@ Studio.moveSingle = function (opts) {
   }
 
   if (skin.gridAlignedMovement) {
+    sprite.setActivity(constants.BEHAVIOR_GRID_ALIGNED);
```

Comment:
I'd love to solicit some opinions on whether or not this is the correct place to set this behavior